libcloud/compute/types.py

```diff
@@ -221,6 +221,18 @@ def fromstring(cls, value):
         return getattr(cls, value.upper(), None)
 
 
+class StorageVolumeState(object):
+    UNKNOWN = None
```

Comment:
Should probably also use a number for `unknown` - that's what other ENUM classes to and this way we can differentiate between "no state" and "state not recognized by libcloud".